services/github/server-github-resources/src/index.ts

```diff
@@ -98,10 +98,48 @@ export async function OnProjectChanges (txes: Tx[], control: TriggerControl): Pr
   return result
 }
 
+/**
+ * @public
+ */
+export async function OnProjectRemove (txes: Tx[], control: TriggerControl): Promise<Tx[]> {
+  const result: Tx[] = []
+  for (const ltx of txes) {
+    if (ltx._class === core.class.TxRemoveDoc) {
+      const cud = ltx as TxCUD<Doc>
+      if (control.hierarchy.isDerived(cud.objectClass, tracker.class.Project)) {
+        const project = control.removedMap.get(cud.objectId)
+        if (project === undefined) {
+          continue
+        }
+        if (control.hierarchy.hasMixin(project, github.mixin.GithubProject)) {
+          const repos = await control.findAll(control.ctx, github.class.GithubIntegrationRepository, {
+            project: cud.objectId as Ref<GithubProject>
+          })
+          for (const repo of repos) {
```

Comment:
Nope, repo should not be deleted, it need to be unlinked.

Reply:
Fixed